hearthstone/entities.py

```diff
@@ -70,6 +71,8 @@ def register_entity(self, entity):
 		self.entities.append(entity)
 		if isinstance(entity, Player):
 			self.players.append(entity)
+		if not self.setup_done:
+			self.initial_entities.append(entity.id)
```

Comment:
You mean filter here already? The problem is that we don't have any tags associated with the entities at this stage yet.